charts/fluentd-elasticsearch/values.yaml

```diff
@@ -198,6 +198,11 @@ serviceMonitor:
   port: 24231
   labels: {}
 
+serviceMetric:
+  ## If true, forces the metrics service to be created
```

Comment:
I'd like to get rid of the usage of the word "force" . There is no such thing as a force created resource from an helm chart. 

Whether it's created or not.

Reply:
I've changed this to:

```
serviceMetric:
  ## If true, the metrics service will be created
  ## Alternative to implicit creation through serviceMonitor.enabled
  ##
  enabled: false
```

Hopefully that's appropriate. If you'd like additional changes, please could you suggest how you'd prefer it to be worder to help me out? Cheers!